atlp-rwanda/cabal-bn-fe
# #181339479: Set up circle ci

### **WHAT DOES THIS PR DO**
This PR is for intergrating circleci so it runs our tests.
### **DESCRIPTION OF TASK TO BE COMPLETED**

1. Add circleci's config.yml
### **HOW CAN THIS BE TESTED**

- Clone this branch and push a commit on it you will see circleci starting to run your tests
### **SCREENSHOTS**

- 
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/95901517/165590756-4b4994e7-44ad-4e39-bac2-34e20e031213.png">
